harvest-dataset/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.8.5-alpine
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY harvester.py .
+
+CMD ["python", "harvester.py"]
```

Comment:
A small suggestion is to place CMD above line 8. The code is more likely to change than the CMD, so this will improve the caching rewards

Reply:
That is true. However, I can't remember having seen a Dockerfile where the CMD or ENTRYPOINT wasn't the last line. The caching rewards is going to be very small/negligable, so I think I'd rather put the CMD where (I assume) the reader expects to find it.

Reply:
Leaving as is since PR was approved.